src/pip/_internal/utils/misc.py

```diff
@@ -59,9 +59,9 @@
 except ImportError:
     # typing's cast() isn't supported in code comments, so we need to
     # define a dummy, no-op version.
-    def cast(typ, val):
+    def cast(typ, val):  # type: ignore
```

Comment:
I don't think we should add `ignore` here. (Also, in general I think we should try to avoid adding `ignore` if we can because otherwise it defeats the purpose of the typing.)

Reply:
Also, if you need help finding ways to avoid `ignore`, feel free to ask.